packages/core/config/babel-plugin-redwood-cell.js

```diff
@@ -0,0 +1,70 @@
+module.exports = function ({ types: t }) {
```

Comment:
```suggestion
// This is supposed to wrap a file that's has a suffix of `Cell` in Redwood's `withCell` higher order component.
// The HOC is responsible for the lifecycle methods during a graphQL query.
// The end result of this plugin is something like:
// ```js
// import { withCell } from '@redwoodjs/web'
// export default withCell({ QUERY, Loading, Succes, Failure, Empty, beforeQuery, afterQuery })
// ```
module.exports = function ({ types: t }) {
```